tests/bridge_message_tests.cpp

```diff
@@ -253,7 +253,7 @@ BOOST_FIXTURE_TEST_CASE(test_bridge_errors, bridge_message_tester) try {
   // Close abcd balance
   close("abcd"_n);
 
-  // receiver account is not open
+  // handler account is not open
```

Comment:
Comment should said `receiver` account is not open.

Reply:
fixed